gcc/rust/parse/rust-parse.cc

```diff
@@ -166,34 +167,69 @@ peculiar_fragment_match_compatible (const AST::MacroMatchFragment &last_match,
 	{MATCH_ARROW, COMMA, EQUAL, PIPE, SEMICOLON, COLON, RIGHT_ANGLE,
 	 RIGHT_SHIFT, LEFT_SQUARE, LEFT_CURLY, AS, WHERE}},
        {AST::MacroFragSpec::VIS,
-	{
-	  COMMA,
-	  IDENTIFIER /* FIXME: Other than `priv` */,
-	  LEFT_PAREN,
-	  LEFT_SQUARE,
-	  EXCLAM,
-	  ASTERISK,
-	  AMP,
-	  LOGICAL_AND,
-	  QUESTION_MARK,
-	  LIFETIME,
-	  LEFT_ANGLE,
-	  LEFT_SHIFT,
-	  SUPER,
-	  SELF,
-	  SELF_ALIAS,
-	  EXTERN_TOK,
-	  CRATE,
-	  UNDERSCORE,
-	  FOR,
-	  IMPL,
-	  FN_TOK,
-	  UNSAFE,
-	  TYPEOF,
-	  DYN
-	  // FIXME: Add Non kw identifiers
-	  // FIXME: Add $crate as valid
-	}}};
+	{COMMA,
```

Comment:
Yeah, sadly there's not really a nice way to do this other than listing the keywords. We could maybe clean this up a little bit by having a separate `std::vector` elsewhere in the file. But this is very nitpicky and I think this is good :)